wsdl/ver20/analytics/humanbody.xsd

```diff
@@ -304,7 +307,43 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 		</xs:sequence>
 		<xs:anyAttribute processContents="lax"/> 
 	</xs:complexType>
-	<xs:complexType name="Belonging">
+	<xs:simpleType name="WeaponCategory">
+		<xs:restriction base="xs:string">
+			<xs:enumeration value="Knife"/>
+			<xs:enumeration value="Rifle"/>
+			<xs:enumeration value="Pistol"/>
+			<xs:enumeration value="Revolver"/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="WeaponPose">
+		<xs:restriction base="xs:string">
+			<xs:enumeration value="InHolster"/>
+			<xs:enumeration value="InHand"/>
+			<xs:enumeration value="Aiming"/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:complexType name="Weapon">
+		<xs:sequence>
+			<xs:element name="Category" type="xs:string" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Describe the Category of the Weapon, acceptable values are defined in bd:WeaponCategory.</xs:documentation> 
+				</xs:annotation> 
+			</xs:element>
+			<xs:element name="Color" type="tt:ColorDescriptor" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Describe the Color of the Cart, acceptable values are defined in tt:ColorDescriptor.</xs:documentation> 
```

Comment:
```suggestion
					<xs:documentation>Describe the Color of the Weapon, acceptable values are defined in tt:ColorDescriptor.</xs:documentation> 
```

Reply:
@HansBusch @bsriramprasad this I think would be a welcome suggestion.